t/20_sdaf_deployment_library.t

```diff
@@ -166,15 +166,13 @@ subtest '[replace_tfvars_variables] Test correct variable replacement' => sub {
 
 subtest '[serial_console_diag_banner] ' => sub {
     my $ms_sdaf = Test::MockModule->new('sles4sap::sap_deployment_automation_framework::deployment', no_auto => 1);
-    my @printed_lines;
-    $ms_sdaf->redefine(enter_cmd => sub { push(@printed_lines, $_[0]); return 1; });
+    my $printed_line;
```

Comment:
```suggestion
    my @printed_lines;
    $ms_sdaf->redefine(enter_cmd => sub { push(@printed_lines, $_[0]); return 1; });
```

 I suggest keeping these two to track unexpected multiple calls, then adapt the rest of the new test logic to use array